### DISCUSSION TITLE
references

### DISCUSSION THREAD

**Discussion by taeil** [Berkeley-Data/hpt#15] [Show and tell]
Any reading to share. 
Starting with. Please use each **reply** for grouping 

#### Terminologies 
- [UTM zone](https://en.wikipedia.org/wiki/Universal_Transverse_Mercator_coordinate_system)
- [Unsupervised Deep Learning for Irrigation Detection in Aerial Images](https://drive.google.com/file/d/1_hl-0QMrHEKpB_DN9kW9toONqRnbQIRu/view)
- [multispectral image](https://en.wikipedia.org/wiki/Multispectral_image)
- [what temperature parameter is](https://www.quora.com/What-is-the-temperature-parameter-in-deep-learning) 
- [semantic segmentation with CRF-RNN](https://www.cs.toronto.edu/~tingwuwang/semantic_segmentation.pdf)
- [Sentinel-2](https://en.wikipedia.org/wiki/Sentinel-2)
- [Latent space](https://towardsdatascience.com/understanding-latent-space-in-machine-learning-de5a7c687d8d#:~:text=The%20latent%20space%20is%20simply,representations%20of%20data%20for%20analysis.) 
- [t-SNE](https://towardsdatascience.com/an-introduction-to-t-sne-with-python-example-5a3a293108d1)
- K-BranchCNN.zip: [http://bigearth.net/static/pretrained-models/BigEarthNet-S2\_19-Classes/K-BranchCNN.zip](http://bigearth.net/static/pretrained-models/BigEarthNet-S2_19-Classes/K-BranchCNN.zip) (from page [https://gitlab.tubit.tu-berlin.de/rsim/BigEarthNet-S2\_19-classes\_models](https://gitlab.tubit.tu-berlin.de/rsim/BigEarthNet-S2_19-classes_models))
- [spectral cheatsheet](https://gisgeography.com/spectral-signature/)  
- [imbalanced data](https://developers.google.com/machine-learning/data-prep/construct/sampling-splitting/imbalanced-data)
- [AUC-ROC](https://towardsdatascience.com/understanding-auc-roc-curve-68b2303cc9c5#:~:text=AUC%20%2D%20ROC%20curve%20is%20a,capable%20of%20distinguishing%20between%20classes.) 
- [GeoTiff](https://trac.osgeo.org/geotiff/) 
- [TFRecord](https://medium.com/mostly-ai/tensorflow-records-what-they-are-and-how-to-use-them-c46bc4bbb564).
- [pretext task vs downstream task](https://stats.stackexchange.com/questions/404602/pretext-task-in-computer-vision)

**Comment by taeil** [comment]
Self-Supervised Learning related readings

**Reply by taeil** [reply]
- [Contrastive Self-Supervised Learning](https://ankeshanand.com/blog/2020/01/26/contrative-self-supervised-learning.html)
- [SimCLR](https://arxiv.org/pdf/2002.05709.pdf)
- [SimCLRv2](https://arxiv.org/pdf/2006.10029.pdf)
- [SimCLR Video](https://www.youtube.com/watch?v=pDJx8i3jenA&list=PLaMu-SDt\_RB4k8VXiB3hOdsn0Y3GoXo1k&index=4)
- [A Survey on Contrastive Self-supervised Learning](https://arxiv.org/abs/2011.00362)
- [A Framework For Contrastive Self-Supervised Learning And Designing A New Approach](https://towardsdatascience.com/a-framework-for-contrastive-self-supervised-learning-and-designing-a-new-approach-3caab5d29619)
- [BigEarthNet dataset description](http://bigearth.net/static/documents/BigEarthNet_IGARSS_2019.pdf)

**Comment by taeil** [comment]
1. [Understand generative learning (not contrastive learning but can get a sense](https://cdn.openai.com/papers/Generative_Pretraining_from_Pixels_V2.pdf)
2. [Contrastive Learning (Augmentations)](https://arxiv.org/pdf/2010.13902.pdf)
3. [Contrastive Learning (Augmentations )](https://openreview.net/pdf?id=KJSC_AsN14)

**Comment by taeil** [comment]
Keras, Tensorflow2, wandb

**Reply by taeil** [reply]
[TensorFlow 2.0 + Keras Overview for Deep Learning Researchers](https://github.com/wandb/examples/blob/master/examples/keras/keras-tensorflow2/TensorFlow_2_0_%2B_Keras_Crash_Course_%2B_W%26B.ipynb)

**Comment by tchken** [comment]
### BE data SOTA relevant 

1. [Pre-trained model](https://tfhub.dev/google/collections/remote_sensing/1)
2. [Intro slide](https://ai4earthscience.github.io/iclr-2020-workshop/papers/ai4earth10slides.pdf)
3. [Reference paper](https://arxiv.org/abs/1911.06721v1)

**Reply by taeil** [reply]
- [Deep learning in remote sensing applications: A meta-analysis and review](https://www.sciencedirect.com/science/article/pii/S0924271619301108)
- BigEarthNet-S2\_tools: [https://gitlab.tubit.tu-berlin.de/rsim/BigEarthNet-S2\_tools](https://gitlab.tubit.tu-berlin.de/rsim/BigEarthNet-S2_tools) (old link: [https://gitlab.tubit.tu-berlin.de/rsim/BigEarthNet-S2\_tools/tree/4a914fa357fd1122741b6976e6c276cdf45b9431](https://gitlab.tubit.tu-berlin.de/rsim/BigEarthNet-S2_tools/tree/4a914fa357fd1122741b6976e6c276cdf45b9431))

**Comment by taeil** [comment]
Multi-source domain adaptation 

- [Geography-Aware Self-supervised Learning Paper](https://arxiv.org/pdf/2011.09980.pdf)
- [Functional Map of the World (FMoW)](https://arxiv.org/abs/1711.07846)

**Comment by taeil** [comment]
[CSL implementation](https://pytorch-lightning-bolts.readthedocs.io/en/latest/self_supervised_models.html#moco-v2)

**Comment by taeil** [comment]
Visualizing GeoTiff images

- [Reading and Visualizing GeoTiff | Satellite Images with Python](https://towardsdatascience.com/reading-and-visualizing-geotiff-images-with-python-8dcca7a74510)
-
